website/events/payables.py

```diff
@@ -24,12 +25,17 @@ def payment_notes(self):
     def payment_payer(self):
         return self.model.member
 
-    def can_create_payment(self, member):
-        return is_organiser(member, self.model.event)
+    def can_manage_payment(self, member):
+        return is_organiser(member, self.model.event) and member.has_perm(
+            "events.change_eventregistration"
```

Comment:
or when it has the override permission or when it is superuser. You could use the service that already exists implementing this check

Reply:
You mean the `is_organiser` check that it is using already?

Reply:
Wut, I'm really blind I guess